Commands/Admin/RemoveTenantSite.cs

```diff
@@ -57,16 +57,16 @@ protected override void ExecuteCmdlet()
                 Url = $"{uri.ToString().TrimEnd('/')}/{Url.TrimStart('/')}";
             }
 
-            bool dodelete = true;
+            bool doDelete = true;
             // Check if not deleting the root web
             var siteUri = new Uri(Url);
             if ($"{siteUri.Scheme}://{siteUri.Host}".Equals(Url, StringComparison.OrdinalIgnoreCase) && !Force)
             {
-                dodelete = false;
-                dodelete = ShouldContinue("You are trying to delete the root site collection. Be aware that you need to contact Office 365 Support in order to create a new root site collection. Also notice that some CSOM and REST operations require the root site collection to be present. Removing this site can affect all your remote processing code, even when accessing non-root site collections.", Resources.Confirm);
+                doDelete = false;
```

Comment:
I'm not sure this line is needed, but maybe there is an edge case with `ShouldContinue`

Reply:
the edge case is that one in principle never should delete the root site, unless you have a -really- good reason to. Hence the question.

Reply:
I mean, is there a need to set the false, since the question in the next line is setting the value to true/false based on ShouldContinue